json_serialization/reader.nim

```diff
@@ -50,23 +50,27 @@ func valueStr(err: ref IntOverflowError): string =
     result.add '-'
   result.add($err.absIntVal)
 
+template tryFmt(expr: untyped): string =
+  try: expr
+  except CatchableError: ""
```

Comment:
an alternative that helps debugging is to return the `msg` on error, in case someone wrote a `$` that raises

Reply:
Addressed here:
https://github.com/status-im/nim-json-serialization/commit/463a0cf1b79f5e586b537c0ce7518e2d1ee85984